src/Kestrel.Core/Internal/Http2/Http2Connection.cs

```diff
@@ -606,7 +601,7 @@ private Task ProcessRstStreamFrameAsync()
             return Task.CompletedTask;
         }
 
-        private async Task ProcessSettingsFrameAsync()
+        private Task ProcessSettingsFrameAsync(ReadOnlySequence<byte> payload)
```

Comment:
part of this was done in https://github.com/aspnet/KestrelHttpServer/commit/de5ccb5c78db1cfd13b93579738be6d80e0ecb08